Somfy/10_SOMFY.pm

```diff
@@ -391,18 +391,28 @@ sub SOMFY_SendCommand($@)
 	# Ys_key_ctrl_cks_rollcode_a0_a1_a2
 	# Ys ad 20 0ae3 a2 98 42
 
-	my $enckey = uc(ReadingsVal($name, "enc_key", "A0"));
-	my $rollingcode = uc(ReadingsVal($name, "rolling_code", "0000"));
-
 	if($command eq "XX") {
 		# use user-supplied custom command
 		$command = $args[1];
 	}
 
+	# increment before sending
```

Comment:
SOMFY_SendCommand: just changed the order of operation. don't increase enc key and rolling code in the end of last sending.